contracts/interfaces/IInterchainTokenService.sol

```diff
@@ -241,23 +241,6 @@ interface IInterchainTokenService is
         uint256 gasValue
     ) external payable;
```

Comment:
update `interchainTransfer` doc to show how to encode `metadata` to call a contract